### DISCUSSION TITLE
How to change the color of the unfilled slice of progress circular?

### DISCUSSION THREAD

**Discussion by pedr0-fr** [vuetifyjs/vuetify#12126] [General]
I want to change this color:

![image](https://user-images.githubusercontent.com/16596928/91339351-43809180-e7ce-11ea-8615-b5f4659b0314.png)

I have read the [component's documentation](https://vuetifyjs.com/en/components/progress-circular/#progress-circular) and tried some CSS attributes but didn't manage to do it.

**Comment by icleolion** [comment]
Two options:

1) override the stroke property of the v-progress-circular__underlay class
![image](https://user-images.githubusercontent.com/47498402/91362807-23af9480-e7f3-11ea-9d0d-6c2c38b4adb3.png)

2) if you are comfortable working with [SASS variables](https://vuetifyjs.com/en/customization/sass-variables/#variable-api), you can override $progress-circular-underlay-stroke as listed in the API for [v-progress-circular](https://vuetifyjs.com/en/components/progress-circular/#progress-circular)
![image](https://user-images.githubusercontent.com/47498402/91363156-e7c8ff00-e7f3-11ea-8049-b223760eb7fe.png)
